pandas/core/arrays/sparse/accessor.py

```diff
@@ -308,8 +308,10 @@ def to_coo(self):
     @property
     def density(self) -> float:
         """
-        Ratio of non-sparse points to total (dense) data points
-        represented in the DataFrame.
+        Ratio of non-sparse points to total (dense) data points.
+
+        Ratio of non-sparse points to total (dense) data points represented
```

Comment:
I think you can remove this extended summary; it says the same thing as the first line